### DISCUSSION TITLE
How do I get my token verified on Phantom Wallet?

### DISCUSSION THREAD

**Discussion by ChoctopusCrypto** [phantom/docs#326] [Help & Support] [answered]
### Summary

How to I verify my token to remove the message "This toke nis unverified. Only interact with tokens you trust?" 

### Steps to Reproduce

go into phantom wallet and search for Choctopus with this CA and the alert pops up. 

EVrGfAj99Xr1NjqqZv6E2msVKZVmhGaThHhcZXzrpump

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Please have your token added to the Jupiter verified list.

**Comment by ChoctopusCrypto** [comment]
Working on it.
